deps.go

```diff
@@ -0,0 +1,236 @@
+package main
+
+import (
+	"bytes"
+	"encoding/json"
+	"io"
+	"os"
+	"os/exec"
+	"path/filepath"
+	"sort"
+	"strings"
+	"time"
+
+	"gopkg.in/errgo.v2/fmt/errors"
+)
+
+func deps(pkgNames []string, withTests bool) (build, test []*Package, err error) {
+	if len(pkgNames) == 0 {
+		// The "no packages means the current package" behaviour
+		// can be done at a higher level.
+		return nil, nil, nil
+	}
+	args := []string{"list", "-deps", "-json"}
+	if withTests {
+		args = append(args, "-test")
+	}
+	args = append(args, pkgNames...)
+	cmd := exec.Command("go", args...)
+	cmd.Stderr = os.Stderr
+	var buf bytes.Buffer
+	cmd.Stdout = &buf
```

Comment:
IMO this could be a bit clearer if you just used `cmd.Output()`

Reply:
done